src/App.js

```diff
@@ -321,38 +330,68 @@ const TabNavigator = () => {
  */
 const AppStack = () => {
   const Stack = createStackNavigator();
+  const [edges, setEdges] = useState([]);
+  const route = useRoute();
+
+  /*
+   * On iOS there are some screens that are not displayed within the "BottomTabBar" context AND have
+   * an interaction element on the bottom of the visible area. When these two conditions happen, it
+   * is more visually comfortable to add a bottom safe area, using the `SafeAreaView` inset
```

Comment:
Typo?
```suggestion
   * is more visually comfortable to add a bottom safe area, using the `SafeAreaView` insert
```

Reply:
Good observation! But the term is actually this one: it refers to an internal padding [on the lib's documentation](https://github.com/th3rdwave/react-native-safe-area-context/#safeareaview).